.github/workflows/gradle_build_and_upload.yml

```diff
@@ -42,12 +42,8 @@ jobs:
 
       - name: Gradle Upload Archives to Snapshot Repository
         if: ${{ !inputs.release }}
-        run: gradle uploadArchives -Psigning.secretKeyRingFile=$GITHUB_WORKSPACE/secring.gpg -PnexusUsername=$NEXUS_UNAME -PnexusPassword=$NEXUS_PWD -Psigning.keyId=$SIGNING_KEYID -Psigning.password=$SIGNING_PWD
+        run: ./gradlew publishVaultClientPublicationToNexusSnapshotsRepository -Psigning.secretKeyRingFile=$GITHUB_WORKSPACE/secring.gpg -PnexusSnapshotsUsername=$NEXUS_UNAME -PnexusSnapshotsPassword=$NEXUS_PWD -Psigning.keyId=$SIGNING_KEYID -Psigning.password=$SIGNING_PWD
 
       - name: Gradle Upload Archives to Releases Repository
         if: ${{ inputs.release }}
-        run: gradle uploadArchives -Prelease -Psigning.secretKeyRingFile=$GITHUB_WORKSPACE/secring.gpg -PnexusUsername=$NEXUS_UNAME -PnexusPassword=$NEXUS_PWD -Psigning.keyId=$SIGNING_KEYID -Psigning.password=$SIGNING_PWD
-
-      - name: Gradle Publish Release to Nexus
-        if: ${{ inputs.release }}
-        run: ./gradlew closeAndReleaseRepository -Psigning.secretKeyRingFile=$GITHUB_WORKSPACE/secring.gpg -PnexusUsername=$NEXUS_UNAME -PnexusPassword=$NEXUS_PWD -Psigning.keyId=$SIGNING_KEYID -Psigning.password=$SIGNING_PWD
```

Comment:
closeAndReleaseRepository command is deprecated in new Gradle version, comment it for now as we can still release it through Nexus UI manually, but need to find an alternative.